roles/traefik/tasks/main.yml

```diff
@@ -8,6 +8,29 @@
     - traefik
     - bootstrap
 
+- name: create directory for local traefik service override
+  sudo: yes
+  file:
+    dest:  /etc/systemd/system/traefik.service.d
+    state: directory
+    mode: 0755
+  tags:
+    - traefik
+    - bootstrap
+
+- name: create local traefik service override
+  sudo: yes
+  copy:
+    dest: /etc/systemd/system/traefik.service.d/marathon.conf
+    content: |
+      [Service]
+      ExecStartPre=-/usr/bin/smlr {{ traefik_marathon_endpoint }}/v2/leader
```

Comment:
it definitely should! good catch, the `-` was masking that problem for me and I must have gotten "lucky" in my testing and didn't run into the #1073 problem. fix coming